examples/script_nifti.py

```diff
@@ -1,10 +1,37 @@
+import os
+import pathlib
+import shutil
+import sys
 from shimmingtoolbox.dicom_to_nifti import dicom_to_nifti
+import tempfile
+import urllib.request as ur
+from zipfile import ZipFile
+
 
 def main():
+    scriptPath = pathlib.Path(__file__).parent.absolute()  # Folder where this script is located
+    dataPath = os.path.join(scriptPath, 'data_testing')  # Folder where the data will be downloaded
+
+    # Download data when not already present
+    if not os.path.isdir(dataPath):
+        url = "https://osf.io/7d2j5/download?versi"
+        try:
+            with ur.urlopen(url) as resp, open("osf_data.zip", 'wb') as out_file:
+                shutil.copyfileobj(resp, out_file)
+                with ZipFile("osf_data.zip", 'r') as zipObj:
+                    # Extract all the contents of zip file in current directory
+                    zipObj.extractall(scriptPath)
+        except IndexError:
+            print("ERROR - {0}:{1}".format(sys.exc_info()[0], sys.exc_info()[1]))
+
+    unsortedDicomDir = os.path.join(dataPath, 'dicom_unsorted')  # Path to the unsorted dicoms
+    # Create temporary folder for processing
+    tmp = tempfile.TemporaryDirectory()
+    niftiPath = os.path.join(tmp.name, 'niftis')  # Path where the niftis will be temporarily stored
 
-    unsortedDicomDir = r"C:\Users\gaspa\Desktop\Test\dicom_unsorted"
-    niftiPath = r"C:\Users\gaspa\Desktop\Test\nifti"
     dicom_to_nifti(unsortedDicomDir, niftiPath)
+    print('Conversion from dicom to nifti done')
```

Comment:
If you write this as

```suggestion
        with tempfile.TemporaryDirection() as tmp:
           niftiPath = niftiPath = os.path.join(tmp.name, 'niftis')
           dicom_to_nifti(unsortedDicomDir, niftiPath)
            print('Conversion from dicom to nifti done')
            # ...
            # further processing ...

then your data will be cleaned up automatically at the end. The `with` statement is python's form of [RAII](https://en.wikipedia.org/wiki/Resource_acquisition_is_initialization).